packages/Filter/src/Filter.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+
+import cx from 'classnames';
+import './filter.scss';
+
+export interface FilterProps extends React.ComponentProps<'button'> {
+  /** Add the main text/icon to the filter */
+  children: React.ReactNode;
+  /** Add class names to the filter button */
```

Comment:
Elsewhere we just call it `filter` (vs `filter button`) for consistency I think the `button` should be remove from the sentence.